### PR TITLE
Lps 152440 5 13

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-152440

/cc @dantewang @tinatian

### CODE REVIEW

**Comment by dantewang** [comment]
ci:test:relevant

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/content/compatibility/converter/JournalContentCompatibilityConverterImpl.java [hunk 1]
```diff
@@ -220,7 +220,7 @@ private String _convertDDMFieldValue(
 	private String _convertLinkToLayoutValue(
 		Locale defaultLocale, String value) {
 
-		if (JSONUtil.isValid(value)) {
+		if (Validator.isNotNull(JSONUtil.createJSONObject(value))) {
```
Comment (dantewang) [inline modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/content/compatibility/converter/JournalContentCompatibilityConverterImpl.java:223]: @slnn Is this SF's requirement to use `Validator.isNotNull`?

**Comment by dantewang** [comment]
@zhangyan0701 Could you review the test results? Thanks!

File: modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/content/compatibility/converter/JournalContentCompatibilityConverterImpl.java (hunk 1 above)
Comment (slnn) [reply modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/content/compatibility/converter/JournalContentCompatibilityConverterImpl.java:223]: No

Comment (slnn) [reply modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/content/compatibility/converter/JournalContentCompatibilityConverterImpl.java]: @dantewang 

Force pushed

**Review by dantewang — APPROVED** [review]

**Comment by tinatian** [comment]
ci:test:relevant

**Comment by tinatian** [comment]
ci:test

**Comment by tinatian** [comment]
ci:close

**Pull request closed without merging**